src/components/date-time-picker/day-view/day-view.component.html

```diff
@@ -19,7 +19,7 @@
                         [class.range-end]="isRangeEndDate(item.date)"
                         [focusIf]="shouldFocus(item)"
                         [attr.aria-label]="item.date | date"
-                        [attr.aria-selected]="isItemActive(item.date, item.isActive)"
```

Comment:
`aria-selected` is fine to use on an element with role="gridcell". I guess the issue here is that the role is on the parent element (`td`) here.

https://www.w3.org/TR/wai-aria/#aria-selected

The date picker buttons are not toggle buttons, so I don't think `aria-pressed` is correct.